portal/integrated-windows-authentication/README.md

```diff
@@ -1,47 +1,41 @@
 
 ## How it works
 
-1. Set up an `AuthenticationChallengeHandler` that challenges the user for authentication, such as the custom handler created for this sample: `IWAChallengeHandler`.
-2. Set the `AuthenticationManager` to use the created challenge handler using `.setAuthenticationChallengeHandler(new IWAChallengeHandler());`.
-3. Create a new `Portal` from a portal URL and a boolean indicating if it is a secured resource using `new Portal(url, true)`.
-4. Load the portal. The authentication challenge handler will prompt the user for credentials. The portal will load successfully with valid credentials.
-5. Create `PortalQueryParameters` to define a search query for map items in the portal.
-5. Find web map portal items using `Portal.findItemsAsync(portalQueryParameters)`.
-6. Create a map using a `PortalItem` result with `new ArcGISMap(portalItem)`.
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username, password, and domain) if a secure resource is encountered.
+2. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates an `UserCredential` object from the information entered by the user.
+3. If the user authenticates, the search returns a list of web map `PortalItem`s and the user can select one to display as an `ArcGISMap`.
 
 ## Relevant API
 
-* AuthenticationChallenge
-* AuthenticationChallengeHandler
-* AuthenticationChallengeResponse
 * AuthenticationManager
 * Portal
 * UserCredential
 
 ## About the data
 
-Searching a public portal with this sample retrieves publicly available web maps from www.arcgis.com.
+This sample searches for web map portal items on a secure portal. To successfully run the sample, you need:
+ * Access to a portal secured with Integrated Windows Authentication that contains one or more web map items.
+ * A login that grants you access to the portal.
 
 ## Additional information
 
 More information about IWA and its use with ArcGIS can be found at the following links:
-
-* [Use Integrated Windows Authentication with your portal](http://enterprise.arcgis.com/en/portal/latest/administer/windows/use-integrated-windows-authentication-with-your-portal.htm)
-* [IWA - Wikipedia](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication)
+ * [IWA - Wikipedia](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication)
+ * [Use Integrated Windows Authentication with your portal](http://enterprise.arcgis.com/en/portal/latest/administer/windows/use-integrated-windows-authentication-with-your-portal.htm)
 
 ## Tags
 
-authentication, IWA, login, portal, security, sign-in
\ No newline at end of file
+authentication, security, Windows
```

Comment:
I an see why we dropped the `IWA` tag, but what about the other ones?

Reply:
not too sure why but to me these more streamlined to IWA. sign-in and login could be for anything, not necessarily IWA related. And I guess the same for portal